environment.yml

```diff
@@ -10,7 +10,7 @@ dependencies:
   - r=4.1.0
   - r-rmarkdown=2.11
   - r-ggplot2=3.3.5
-  - r-dplyr
+  - r-dplyr=1.0.9
```

Comment:
I had to update this dependency because of the following error 

```

Command error:
  Loading required package: rmarkdown
  Warning message:
  package 'rmarkdown' was built under R version 4.1.2 
  
  
  processing file: gwas_report_template.Rmd
  Quitting from lines 30-39 (gwas_report_template.Rmd) 
  Error: package or namespace load failed for 'ramwas' in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]):
   namespace 'dplyr' 1.0.7 is already loaded, but >= 1.0.9 is required
  Execution halted



```